support/k8s/k8s-workload-registrar/README.md

```diff
@@ -22,49 +22,50 @@ The configuration file is a **required** by the registrar. It contains
 
 | Key                        | Type    | Required? | Description                              | Default |
 | -------------------------- | --------| ---------| ----------------------------------------- | ------- |
-| `log_level`                | string  | required | Log level (one of `"panic"`,`"fatal"`,`"error"`,`"warn"`, `"warning"`,`"info"`,`"debug"`,`"trace"`) | `"info"` |
+| `log_level`                | string  | optional | Log level (one of `"panic"`,`"fatal"`,`"error"`,`"warn"`, `"warning"`,`"info"`,`"debug"`,`"trace"`) | `"info"` |
 | `log_path`                 | string  | optional | Path on disk to write the log | |
-| `addr`                     | string  | required | Address to bind the HTTPS listener to | `":8443"` |
-| `cert_path`                | string  | required | Path on disk to the PEM-encoded server TLS certificate | `"cert.pem"` |
-| `key_path`                 | string  | required | Path on disk to the PEM-encoded server TLS key |  `"key.pem"` |
-| `cacert_path`              | string  | required | Path on disk to the CA certificate used to verify the client (i.e. API server) | `"cacert.pem"` |
-| `insecure_skip_client_verification` | boolean | required | If true, skips client certificate verification (in which case `cacert_path` is ignored). See [Security Considerations](#security-considerations) for more details. | `false` |
+| `metrics_addr`             | string  | optional | Address to expose metrics on, use `0` to disable. | 0 |
+| `agent_socket_path`        | string  | optional | Path to the Unix domain socket of the SPIRE agent. If not specified, attempt to connect to the server insecurely | |
+| `server_address`           | string  | required | Address of the spire server | |
 | `trust_domain`             | string  | required | Trust domain of the SPIRE server | |
-| `server_socket_path`       | string  | required | Path to the Unix domain socket of the SPIRE server | |
```

Comment:
Not really since you can use `server_addr = "unix:///path/to/socket"`. In refactoring on top of master I've reintroduced the docs for `server_socket_path`.